src/Security/Validation/AbstractPasswordValidator.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace SilverStripe\Security\Validation;
+
+use SilverStripe\Core\Config\Configurable;
+use SilverStripe\Core\Injector\Injectable;
+use SilverStripe\ORM\DataObject;
+use SilverStripe\Core\Validation\ValidationResult;
+use SilverStripe\Security\Member;
+use SilverStripe\Security\MemberPassword;
+
+/**
+ * Abstract validator with functionality for checking for reusing old passwords.
+ */
+abstract class PasswordValidator
+{
+    use Injectable;
+    use Configurable;
```

Comment:
```suggestion
    use Injectable;
    use Configurable;
    use Extensible;
```
Will also need to add the import

Reply:
I assume you mean for extensible....?